tsconfig.json

```diff
@@ -1,10 +1,14 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "nodenext",
+    // If we were just compiling for the tests, we could safely target ESNext at
+    // all times, but just because we've been starting to add more runtime logic
```

Comment:
We only have the runtime logic in the windows-rdp module, right?

Reply:
Right now, yeah

Reply:
But one of the other benefits of having an older compilation target is that, because I have that JavaScript file set up to be part of the TypeScript project, the TypeScript LSP can catch any function calls that might be using features that are too recent